### DISCUSSION TITLE
The specified pegel identifier is invalid.

### DISCUSSION THREAD

**Discussion by Jo-Kaan** [stephan192/hochwasserportal#23] [Q&A]
Hallo und einen schönen guten Abend, 
ich habe das Hochwasserportal jetzt über die Integration geladen und die Einträge in der configuration.yaml entfernt. Bei Aktivierung der Integration wird nach Eingabe der Pegel ID die Fehlermeldung: "The specified pegel identifier is invalid." ausgegeben. Ich habe verschiedene Pegel IDs aus der pegel.md probiert, erhalte jedoch immer diesen Fehler.
Mache ich etwas falsch oder ist ein Fehler in der Integration?

Viele Grüße
Jo

**Comment by stephan192** [comment]
Welche Pegel funktionieren nicht? Bitte immer IDs angeben!
**Wie auf der Startseite angegeben funktionieren aktuell nur Pegel die mit BY, NI, NW und SH beginnen.**
siehe auch Issue #12
Ich habe gerade eben die Pegelliste aktualisiert.

**Reply by Jo-Kaan** [reply]
Vielen Dank für die schnelle Antwort. 
Sorry, den Hinweis hatte ich nicht gelesen.
Als ich die IDs aus der Pegelliste genommen habe, waren  noch SN, ST und TH vorhanden. Speziell benötige ich SN und ST.

**Reply by Jo-Kaan** [reply]
Hallo, 
wie ich soeben gesehen habe, ist die Pegelliste neu eingestellt. Dabei ist mir aufgefallen, dass die Stationen von Sachsen und Sachsen-Anhalt alle unter ST_ geführt werden. Sicher ein Versehen?
Auch funktionieren diese Pegel IDs nicht, z.B. ST_501060 (Dresden, Sachsen).

**Reply by Jo-Kaan** [reply]
Sorry, ST_ . Ich habe meinen Kommentar korrigiert.

**Reply by stephan192** [reply]
Ich frage das Portal von Sachsen-Anhalt nach allen Pegeln die sie zur Verfügung stellen und anscheinend liefert deren Portal auch Werte für Sachsen. Daher bekommen sie ST_ vorangestellt, das läuft alles automatisch, da wird nichts händisch angepasst.

Zum Pegel ST_501060. Der funktioniert bei mir ohne Probleme und liefert level, stage und flow.

**Reply by stephan192** [reply]
Die ST und TH Pegel funktionieren natürlich nur mit der neusten Version v0.0.7 der Intergration!

**Reply by Jo-Kaan** [reply]
Okay, danke für die Informationen. Eine Aktualisierung auf die neue Version wurde mir bisher nicht angezeigt.

**Reply by Jo-Kaan** [reply]
Hallo und vielen Dank. Nach der Aktualisierung auf Version v0.0.7 sind meine gewünschten Pegel wieder verfügbar und werden angezeigt :-)

**Reply by wallo66** [reply]
@stephan192  vielen für die neue Integration der RLP Pegelwerte! Ich wollte schon per Pythonscript die für mich interessanten Werte scrapen
